executor/adapter.go

```diff
@@ -27,12 +27,17 @@ import (
 	"github.com/pingcap/tidb/plan"
 )
 
+type processinfoReset interface {
+	SetProcessInfo(string)
```

Comment:
`processinfoResetter` is more like an interface name.
And the method can be `ResetProcessInfo` without string argument.